terraform/modules/efs/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_efs_file_system" "efs" {
```

Comment:
I think we can follow the resource naming convention changes done @roperto in other modules